bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/Configuration.kt

```diff
@@ -89,7 +89,7 @@ data class CaptureConfiguration(
     /**
      * Set whether the camera flash is on or off by default.
```

Comment:
[Suggestion] What do you think of repharsing it to:
```suggestion
     * Set whether the camera flash should be on or off when the SDK starts. The flash is off by default.
```

Reply:
Done! Also changed for capture docs :-)